cimr/processor/convertibles.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+"""Utility functions to convert between values and units
+for downstream analyses.
+e.g. log(OR) -> effect_size
+"""
+
+__author__ = "yoson park"
+
+
+import re
+import numpy
+import pandas
+import logging
+
+from scipy import stats
+
+from .constants import (EFFECT_SIZE, ODDS_RATIO, ZSCORE,
+    PVALUE, STANDARD_ERROR, EFFECT_DIRECTION)
+
+from ..defaults import VERY_SMALL_P
+
+
+def get_effect_direction(data):
+    """Distinguish datasets with absolute beta effects only.
+    data is assumed to be a pandas dataframe
+    """
+    effect_direction = None
+
+    if EFFECT_SIZE in data:
+        logging.debug(f' checking direction of effects.')
+        effect_direction = numpy.sign(data[EFFECT_SIZE])
+    elif EFFECT_DIRECTION in data:
+        logging.debug(f' checking effect_dir column.')
+        effect_direction = data[EFFECT_DIRECTION]
+        # Make it equivalent to numpy.sign
+        effect_direction = effect_direction.apply(
+            lambda x: 1.0 if (x == '+' or x==1.0) else -1.0
+        )
+
+    if effect_direction is None:
+        logging.warning(f' effect direction is not provided.')
+    return effect_direction
+
+
+def get_z(data, VERY_SMALL_P):
+    """Given dataset with effect_size and standard_error or
+    pvalue columns, calculate zscore.
```

Comment:
`p-value` is probably better. (This comment applies to all occurrences of `pvalue` in this file.)

Reply:
both `zscore` and `pvalue` are referring to the column name within cimr, but I agree it's better to make it correct in the docstrings